frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/EventsCard.tsx

```diff
@@ -11,31 +11,18 @@ import ActivityBody, {
   Activity,
 } from '@console/shared/src/components/dashboard/activity-card/ActivityBody';
 import ActivityItem from '@console/shared/src/components/dashboard/activity-card/ActivityItem';
-import {
-  EventKind,
-  K8sResourceKind,
-  MachineKind,
-  referenceForModel,
-} from '@console/internal/module/k8s';
+import { EventKind, K8sResourceKind, MachineKind } from '@console/internal/module/k8s';
```

Comment:
did prettier/eslint move this to one line?

Reply:
Yes.